src/components/Author/index.tsx

```diff
@@ -26,13 +26,13 @@ const Author = ({
         <a
           className={styles.link}
           href={githubLink}
-          title={username}
+          aria-label={`${username} Github - opens in new tab`}
```

Comment:
Could we instead of adding explicit English here, add a new entry in our `en.json` file?

So we might be able to translate this text to other languages 👀 (I forgot to mention that for other screen-reader specific content 😅)

Reply:
sure, is it just a case of adding an entry into the json file? Anything special that needs to be done for the interpolation?

Reply:
We have a file you could follow called TRANSLATING.md

Not sure if it covers adding new entries, bt I'm pretty sure I added some stuff there 👀 

If not, let me know and I rush to add a section there 🤓